TestCases/rans/naca0012/turb_NACA0012_sst_sust.cfg

```diff
@@ -18,7 +18,8 @@
 SOLVER= RANS
 %
 % Specify turbulent model (NONE, SA, SA_NEG, SST)
-KIND_TURB_MODEL= SST_SUST
+KIND_TURB_MODEL= SST
+SST_OPTIONS=(SUSTAINING)
```

Comment:
So what is the plan for the SUST and UQ options? From the code it looks like you want to support both, from the testcases it looks like you want to drop the old stuff?
I don't mind it either way, just wan't to know where we're going before looking at the code.

Reply:
I was first of the opinion that we should have a grace period for keywords that will become obsolete in the future. On the other hand, it is usually not a big deal to change a single keyword in a config file. 
If @WallyMaier agrees, we can ditch the old keywords in its entirety.

Reply:
SUST and UQ are probably not used very often out in the wild, and we would go to 7.4 when this is released, so I think it is fine to drop the old ones.

Reply:
Im on board with completely moving the SUST and UQ over to SST_OPTIONS.  If a big deal, we could put an error saying "SST_SUST no longer an option"

Reply:
We will put that error.